packages/react-router-dom/__tests__/ssr/components-test.tsx

```diff
@@ -342,7 +341,6 @@ describe("<RemixBrowser>", () => {
       url: "",
       version: "",
     };
-    _setSsrInfoForTests(window);
```

Comment:
Changed how this works in `RouterProvider` so we don't need this unit test shim method